Site/SiteAnalyticsModule.php

```diff
@@ -178,7 +178,13 @@ public function init()
 		$this->friendbuy_account_id =
 			$config->analytics->friendbuy_account_id;
 
-		$this->initOptOut();
+		if (!$config->analytics->enabled) {
+			// We don't want to have any analytics on this site
+			// So we'll treat everyone as an opt-out
+			$this->analytics_opt_out = true;
```

Comment:
Rather than re-using the opt_out flag I think we should check enabled in all the same places we check opt_out.